syntaxes/elm.json

```diff
@@ -343,17 +343,17 @@
       ]
     },
     "char": {
-      "name": "char.quoted.elm",
+      "name": "string.quoted.char.elm",
```

Comment:
if we go by naming conventions mentioned here https://macromates.com/manual/en/language_grammars this should be `string.quoted.single.elm`